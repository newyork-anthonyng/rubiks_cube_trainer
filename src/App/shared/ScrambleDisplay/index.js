import React from "react";
import PropTypes from "prop-types";

const headerStyle = {
  textAlign: "center"
};

const ScrambleDisplay = ({ text }) => <h1 style={headerStyle}>{text}</h1>;

ScrambleDisplay.propTypes = {
  text: PropTypes.string
};

export default ScrambleDisplay;
