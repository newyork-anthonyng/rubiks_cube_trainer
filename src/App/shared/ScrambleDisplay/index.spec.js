import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import ScrambleDisplay from "./";

const defaultProps = {
  text: "R U R'"
};

it("should render correctly", () => {
  const wrapper = shallow(<ScrambleDisplay {...defaultProps} />);

  expect(toJSON(wrapper)).toMatchSnapshot();
});
