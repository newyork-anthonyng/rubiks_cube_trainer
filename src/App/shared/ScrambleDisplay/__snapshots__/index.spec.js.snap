// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<h1
  style={
    Object {
      "textAlign": "center",
    }
  }
>
  R U R'
</h1>
`;
